client/terraform_cli_test.go

```diff
@@ -266,6 +268,99 @@ func TestTerraformCLIPlan(t *testing.T) {
 	}
 }
 
+func TestTerraformCLIValidate(t *testing.T) {
+	t.Parallel()
+
+	errJson := `{
+		"valid": false,
+		"diagnostics": [
```

Comment:
Strings in backticks preserve the exact string and its spacing, so here it would have the raw JSON include the go-linter tabs that align it wonky. For example:

```json
{
        "valid": false,
        "diagnostics": [ ]
    }
```

To get the pretty formatted json
```json
{
  "valid": false,
  "diagnostics": [ ]
}
```

The go backtick'ed string would need to resemble
```go
	errJson := `{
  "valid": false,
  "diagnostics": [ ]
}`
```

Since this is a test and the spacing within the raw string itself doesn't matter to the JSON unmarshaller, this is a nit and you don't have to change! it's pretty readable as-is.